src/scripts/main.js

```diff
@@ -1,3 +1,78 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('table');
+const thead = document.querySelector('thead');
+const tbody = table.querySelector('tbody');
+const rows = [...table.querySelectorAll('tbody tr')];
+const headerCells = thead.querySelectorAll('th');
+
+headerCells[0].addEventListener('click', () => {
+  const sortedRows = rows.sort((rowA, rowB) => {
```

Comment:
The `rows.sort()` method sorts the array in place, which means the original order of rows is lost after the first sort. Consider creating a new array for sorting to preserve the original order for subsequent sorts.